cache/.htaccess

```diff
@@ -1 +0,0 @@
-Deny from all
```

Comment:
This isn't intentional, is it?

Reply:
Good catch. This is why it's best to clear your cache by doing a `composer install`. It will leave behind the .htaccess file.